Binary files a/.DS_Store and b/.DS_Store differ
Binary files a/LCAClassifier/.DS_Store and b/LCAClassifier/.DS_Store differ
Only in b/LCAClassifier/etc: .DS_Store
Binary files a/LCAClassifier/src/.DS_Store and b/LCAClassifier/src/.DS_Store differ
Only in b/LCAClassifier/src/LCAClassifier: .DS_Store
diff -u -r a/LCAClassifier/src/LCAClassifier/arbor.py b/LCAClassifier/src/LCAClassifier/arbor.py
--- a/LCAClassifier/src/LCAClassifier/arbor.py	2017-03-07 13:51:40.000000000 +0200
+++ b/LCAClassifier/src/LCAClassifier/arbor.py	2017-03-07 16:41:39.000000000 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
 import sys
 import re
 import os
diff -u -r a/LCAClassifier/src/LCAClassifier/classify.py b/LCAClassifier/src/LCAClassifier/classify.py
--- a/LCAClassifier/src/LCAClassifier/classify.py	2017-03-07 13:51:40.000000000 +0200
+++ b/LCAClassifier/src/LCAClassifier/classify.py	2017-03-07 16:41:47.000000000 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
 '''
 Created on Feb 26, 2016
 
diff -u -r a/LCAClassifier/src/LCAClassifier/config.py b/LCAClassifier/src/LCAClassifier/config.py
--- a/LCAClassifier/src/LCAClassifier/config.py	2017-03-07 13:51:40.000000000 +0200
+++ b/LCAClassifier/src/LCAClassifier/config.py	2017-03-07 16:41:54.000000000 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
 import ConfigParser
 import os.path
 import sys
diff -u -r a/LCAClassifier/src/LCAClassifier/evaluate.py b/LCAClassifier/src/LCAClassifier/evaluate.py
--- a/LCAClassifier/src/LCAClassifier/evaluate.py	2017-03-07 13:51:40.000000000 +0200
+++ b/LCAClassifier/src/LCAClassifier/evaluate.py	2017-03-07 16:42:00.000000000 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
 '''
 Created on Mar 7, 2017
 
diff -u -r a/LCAClassifier/src/LCAClassifier/nds2CREST.py b/LCAClassifier/src/LCAClassifier/nds2CREST.py
--- a/LCAClassifier/src/LCAClassifier/nds2CREST.py	2017-03-07 13:51:40.000000000 +0200
+++ b/LCAClassifier/src/LCAClassifier/nds2CREST.py	2017-03-07 16:42:07.000000000 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
 """Script to convert and NDS file from Silva into a map, synonyms and taxonomy
 file for MEGANs.
 
diff -u -r a/LCAClassifier/src/LCAClassifier/taxa.py b/LCAClassifier/src/LCAClassifier/taxa.py
--- a/LCAClassifier/src/LCAClassifier/taxa.py	2017-03-07 13:51:40.000000000 +0200
+++ b/LCAClassifier/src/LCAClassifier/taxa.py	2017-03-07 16:42:14.000000000 +0200
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
 import sys
 import re
 from collections import deque
